api/mesh/v1alpha1/metrics.proto

```diff
@@ -71,3 +74,16 @@ message PrometheusAggregateMetricsConfig {
   // as true and kuma-dp scrapes metrics from the service.
   google.protobuf.BoolValue enabled = 4;
 }
+
+// PrometheusEnvoyFilterConfig defines filters that should be passed to Envoy
+// for filtering.
+message PrometheusEnvoyFilterConfig {
```

Comment:
I think I'd remove Filter from this name. I can see how we potentially can put other options here